unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -45,6 +45,9 @@ void Awake()
             Debug.unityLogger.logEnabled = false;
 #endif
 
+            // Warmup some shader variants
+            Resources.Load<ShaderVariantCollection>("ShaderVariantCollections/shaderVariants-selected").WarmUp();
```

Comment:
If this call is blocking, this means the kernel messages can start pouring in before the rest of the code in this `Awake()` call, leading to issues. We should move this call at the very least to the end of this method.